app/models/album.rb

```diff
@@ -70,9 +70,12 @@ def album_artist_separators
     album_artists.each do |aa|
       if aa.order == album_artists.to_a.count
         errors.add(:album_artists, 'aa-last-no-separator') unless aa.separator.nil?
-      elsif aa.separator.blank?
+      # rubocop:disable Rails/Blank
+      # We don't want to check for blank? here, since we specifically want to allow a blank string
+      elsif aa.separator.nil? || aa.separator.empty?
```

Comment:
Maybe we should allow the empty string as well? I'm thinking about cases like [NehruvianDOOM](https://www.discogs.com/NehruvianDOOM-NehruvianDOOM/master/741082) where the "Nehruvian" part is "Bishop Nehru" and the "DOOM" part is "MF DOOM".

Reply:
True, I've updated the PR